templates/around/map.html

```diff
@@ -74,11 +74,12 @@ <h2>Share your place</h2>
   <script>
     const usersLocation = []
     {% for user in users %}
-      {% if not current_user %}
+      {% if user != current_user %}
         usersLocation.push({ 
           name: '{{ user.username }}',
```

Comment:
Makes me think that we should escape characters.
I the user enters something malicious it could screw…
Let's say his name is `john's doe`; the page breaks.
But if he's called `john';alert('xss attack')` then we're screwed.

Reply:
We're actually [safe by default](http://flask.pocoo.org/docs/0.12/security/#cross-site-scripting-xss). 😅